index.src.html

```diff
@@ -349,6 +348,54 @@
           view</dfn> which is a double. It represents the vertical angle of
           view in degrees.
         </p>
+        <p>
+          The data type of a <a>depth map</a> is 16-bit unsigned integer. The
+          algorithm to <dfn>convert the depth map value to grayscale</dfn>,
+          given a depth map value <var>d</var>, is as follows:
+        </p>
+        <ol>
+          <li>Let <var>near</var> be the the <a>near value</a>.
+          </li>
+          <li>Let <var>far</var> be the the <a>far value</a>.
+          </li>
+          <li>Apply the <a>rules to convert using range inverse</a> to
+          <var>d</var> to obtain quantized value <var>d<sub>8bit</sub>.</var>
+          </li>
+          <li>Return <var>d<sub>16bit</sub></var>.
+          </li>
+        </ol>
+        <p>
+          The <dfn>rules to convert using range inverse</dfn> are as given in
```

Comment:
Change to range linear.

(Was in an earlier version, see:
https://www.w3.org/TR/2016/WD-mediacapture-depth-20160226/#dfn-rules-to-convert-using-range-linear)